benchmark_runner/common/oc/oc.py

```diff
@@ -1289,6 +1289,24 @@ def get_vm_ssh_status(self, node_ip: str = '', vm_node_port: str = ''):
         else:
             return self.run(ssh_vm_cmd)
 
+    def wait_for_vm_virtctl(self, vm_name: str = '', namespace: str = environment_variables.environment_variables_dict['namespace'],
```

Comment:
Create a function `get_vm_status` that hides the method used to actually do it.  You shouldn't need to change any other code to switch between `ssh` and `virtctl ssh`.

Reply:
Its 2 different approaches, I dont want to have any relation between it.

Reply:
Its two different approaches, but the end result is the same and nothing outside of this file should have any reason to know how we retrieve the status.